shared/src/store/action-reducers/simulation.ts

```diff
@@ -647,21 +649,57 @@ export namespace SimulationActionReducers {
                 const treatPatientsBehaviorState = behaviorStates.find(
                     (behaviorState) => behaviorState.id === behaviorStateId
                 ) as Mutable<TreatPatientsBehaviorState>;
+
                 if (unknown !== undefined) {
                     treatPatientsBehaviorState.intervals.unknown = unknown;
+                    logBehavior(
+                        draftState,
+                        [],
+                        `Das ${treatPatientsBehaviorState.type} Verhalten im Bereich ${simulatedRegion.name} wird in der Phase Erkunden alle ${unknown}ms die Zuteilung neu berechnen`,
```

Comment:
This applies to all log entries in this file: There is a `behaviorTypeToGermanNameDictionary` in the frontend. You can refactor this, to be used here as well. We don't want such type strings in our logs

Reply:
d